languages/Payment/en.ini

```diff
@@ -0,0 +1,41 @@
+; Payment-related translations
+Accept = "Accept"
+breakdown_title = "Payment Receipt"
+Cancel = "Cancel"
+Contact Information = "Contact Information"
+Date = "Date"
+Details = "Details"
+download_receipt_pdf = "Download Receipt (PDF)"
+email_address_missing = "Email address is missing"
```

Comment:
Might this make sense as a top-level string, since it could apply in multiple contexts? I don't see a reason for specifically scoping this to the payment context (though it seems justifiable for all the other lines in the file).